templates/compositions/ec-component-highlighted-news-block/highlighted-news-block.html.twig

```diff
@@ -19,6 +20,14 @@
         </div>
       </div>
     {% endif %}
+    {% set _introduction = introduction|default('') %}
```

Comment:
Why is this needed? Instead of moving the value from a variable to another, we can just check if `introduction` is not empty.

Reply:
I just followed the way from other components, I can change it.

Reply:
Fixed